tests/unit/amo/components/TestAddonReviewList.js

```diff
@@ -214,6 +214,25 @@ describe(__filename, () => {
       }));
     });
 
+    it('dispatches fetchReviews with an invalid page variable', () => {
+      const dispatch = sinon.stub(store, 'dispatch');
+      const errorHandler = createStubErrorHandler();
+      const addonSlug = fakeAddon.slug;
+      const page = 'x';
```

Comment:
Since it's not totally obvious what this test is about I think it's worth adding a comment to say that we pass invalid pages to the API intentionally to get a 404 response.

Reply:
Good idea. Done!